Libplanet.Explorer.Executable/Program.cs

```diff
@@ -60,8 +58,8 @@ public static async Task Main(string[] args)
         public void Schema()
         {
             var serviceCollection = new ServiceCollection();
-            serviceCollection.AddGraphQL()
-                .AddGraphTypes(typeof(LibplanetExplorerSchema<NullAction>));
+            serviceCollection.AddGraphQL(builder =>
+                builder.AddSchema<LibplanetExplorerSchema<NullAction>>());
```

Comment:
`AddSchema` doesn't add GraphQL types automatically. It should call the `AddGraphTypes` manually too.

You can check it by running `dotnet run -- schema` command.

```suggestion
            serviceCollection.AddGraphQL(builder =>
                builder.AddSchema<LibplanetExplorerSchema<NullAction>>()
                    .AddGraphTypes(typeof(LibplanetExplorerSchema<NullAction>).Assembly));
```

Reply:
Thank you for pointing it out!  I'm going to add `.AddGraphTypes()` in `ConfigureLibplanetExplorerSchema.Configure()` instead.

Reply:
Hmm… it apparently cannot be placed in `ConfigureLibplanetExplorerSchema.Configure()`, because `IConfigureSchema` interface does not take `IGraphQLBuilder`.  I'm going to add `AddLibplanetExplorerSchema()` extension method to `IGraphQLBuilder` instead.